doc/tutorials/using_grub.rst

```diff
@@ -11,51 +11,52 @@ latest GRUB source code and
-Which boot protocol is used depends on the hypervisor is loaded by
-GRUB's ``multiboot`` command or ``multiboot2`` command. The guest kernel
-or ramdisk must be loaded by the GRUB ``module`` command or ``module2``
-command accordingly when different boot protocol is used.
+<http://www.gnu.org/software/grub/manual/multiboot2/multiboot.html>`_. Compared
+with the multiboot protocol, the multiboot2 protocol adds UEFI support.
+
+The multiboot protocol is supported by the ACRN hypervisor natively. The
+multiboot2 protocol is supported when :option:`hv.FEATURES.MULTIBOOT2` is
+enabled in the scenario configuration. The :option:`hv.FEATURES.MULTIBOOT2` is
+enabled by default. To load the hypervisor with the multiboot protocol, run the
+GRUB ``multiboot`` command. To load the hypervisor with the multiboot2 protocol,
+run the ``multiboot2`` command. To load a VM kernel or ramdisk, run the
+GRUB ``module`` command for the multiboot protocol or the ``module2`` command
+for the multiboot2 protocol.
 
 The ACRN hypervisor binary is built with two formats: ``acrn.32.out`` in
 ELF format and ``acrn.bin`` in RAW format. The GRUB ``multiboot``
-command support ELF format only and does not support binary relocation,
+command supports ELF format only and does not support binary relocation,
 even if :option:`hv.FEATURES.RELOC` is set. The GRUB ``multiboot2``
 command supports
 ELF format when :option:`hv.FEATURES.RELOC` is not set, or RAW format when
 :option:`hv.FEATURES.RELOC` is set.
 
 .. note::
-   * :option:`hv.FEATURES.RELOC` is set by default, so use ``acrn.32.out`` in multiboot
-     protocol and ``acrn.bin`` in multiboot2 protocol.
+   * :option:`hv.FEATURES.RELOC` is set by default, so use ``acrn.32.out`` in
+     the multiboot protocol and ``acrn.bin`` in the multiboot2 protocol.
 
    * Per ACPI specification, the RSDP pointer is described in the EFI System
-     Table instead of legacy ACPI RSDP area on a UEFI enabled platform. To make
-     sure ACRN hypervisor gets the correct ACPI RSDP info, we recommend using
-     ``acrn.bin`` with multiboot2 protocol to load hypervisor on a UEFI platform.
+     Table instead of the legacy ACPI RSDP area on a UEFI-enabled platform. To
+     make sure the ACRN hypervisor gets the correct ACPI RSDP information, we
+     recommend using ``acrn.bin`` with the multiboot2 protocol to load the
+     hypervisor on a UEFI platform.
 
 .. _pre-installed-grub:
 
 Using Pre-Installed GRUB
 ************************
 
-Most Linux distributions use GRUB version 2 by default. If its version
-2.02 or higher, we can re-use the pre-installed GRUB to load the ACRN
+Most Linux distributions use GRUB version 2 by default. For version
+2.02 or higher, we can reuse the pre-installed GRUB to load the ACRN
 hypervisor.
 
 Here's an example using Ubuntu to load ACRN on a scenario with two
-pre-launched VMs (the SOS_VM is also a kind of pre-launched VM):
+pre-launched VMs (the Service VM is also a kind of pre-launched VM):
 
```

Comment:
Why do we need this text: "the Service VM is also a kind of pre-launched VM"? It may be confusing, because we are trying to distinguish pre-launched VMs from Service VM.